client/ayon_core/plugins/publish/collect_otio_review.py

```diff
@@ -95,9 +95,46 @@ def process(self, instance):
             instance.data["label"] = label + " (review)"
             instance.data["families"] += ["review", "ftrack"]
             instance.data["otioReviewClips"] = otio_review_clips
+
             self.log.info(
                 "Creating review track: {}".format(otio_review_clips))
 
+            # get colorspace from metadata if available
+            if len(otio_review_clips) >= 1 and any(
+                # lets make sure any clip with media reference is found
+                (
+                    clip
+                    for clip in otio_review_clips
+                    if isinstance(clip, otio.schema.Clip)
+                    and clip.media_reference
+                )
```

Comment:
You don't need t ocreate tuple first, you can just use iteration as is.
```suggestion
                clip
                for clip in otio_review_clips
                if (
                    isinstance(clip, otio.schema.Clip)
                    and clip.media_reference
                )
```